pylint/checkers/base/basic_checker.py

```diff
@@ -444,6 +451,12 @@ def visit_expr(self, node: nodes.Expr) -> None:
             self.add_message("pointless-string-statement", node=node)
             return
 
+        # Warn W0133 for exceptions that are used as statements
+        if isinstance(expr, nodes.Call):
+            inferred = utils.safe_infer(expr)
+            if isinstance(inferred, objects.ExceptionInstance):
+                self.add_message("pointless-exception-statement", node=node)
```

Comment:
```suggestion
                self.add_message("pointless-exception-statement", node=node, confidence=INFERENCE)
```
This will require to import it and to regenerate the functional output.